docs/content/preview/architecture/key-concepts.md

```diff
@@ -0,0 +1,137 @@
+
+Read replica clusters can perform only reads; writes sent to read replica clusters get automatically rerouted to the [primary cluster](#primary-cluster) of the [universe](#universe). These clusters enable reads in regions that are far away from the primary cluster with timeline-consistent data. This ensures low latency reads for geo-distributed applications.
+
+Data is brought into the read replica clusters through asynchronous replication from the primary cluster. In other words, [nodes](#node) in a read replica cluster act as Raft observers that do not participate in the write path involving the Raft leader and Raft followers present in the primary cluster. {{<link "../docdb-replication/read-replicas">}}
+
+## Rebalancing
+
+Rebalancing is the process of keeping an even distribution of tablets across the [nodes](#node) in a cluster. {{<link "../../explore/linear-scalability/data-distribution/#rebalancing">}}
+
+## Replication factor (RF)
+
+The number of copies of data in a YugabyteDB universe. YugabyteDB replicates data across zones (or fault domains) in order to tolerate faults. Fault tolerance (FT) and RF are correlated. To achieve a FT of k nodes, the universe has to be configured with a RF of (2k + 1).
+
+The RF should be an odd number to ensure majority consensus can be established during failures. {{<link "../docdb-replication/replication/#replication-factor">}}
+
+## Sharding
+
+Sharding is the process of mapping a table row to a [tablet](#tablet). YugabyteDB supports 2 types of sharding, Hash and Range. {{<link "../docdb-sharding">}}
+
+## Tablet
+
+YugabyteDB splits a table into multiple small pieces called tablets for data distribution. The word "tablet" finds its origins in ancient history, when civilizations utilized flat slabs made of clay or stone as surfaces for writing and maintaining records. {{<link "../../explore/linear-scalability/data-distribution/">}}
+
+{{<tip>}}
+Tablets are also referred as shards.
+{{</tip>}}
+
+## Tablet splitting
+
+When a tablet reaches a threshold size, it splits into 2 new [tablets](#tablet). This is a very quick operation. {{<link "../docdb-sharding/tablet-splitting">}}
+
+## TServer
+
+The [YB-TServer](../yb-tserver) service is responsible for maintaining and managing table data in the form of tablets, as well as dealing with all the queries. {{<link "yb-tserver">}}
+
+## Universe
+
+A YugabyteDB universe comprises one primary cluster and zero or more read replica clusters that collectively function as a resilient and scalable distributed database.
+
+{{<note>}}
+Sometimes the terms *universe* and *cluster* are used interchangeably. However, the two are not always equivalent, as a [Universe](#universe) can contain one or more [clusters](#cluster)
+{{</note>}}
+
+## xCluster
+
+xCluster is the scheme of asynchronous replication between 2 [universes](#universe). Primary used for disaster recovery. YugabyteDB supports transactional xCluster {{<link "../docdb-replication/async-replication/">}}
```

Comment:
Add a docs link. We should also sort out the endless variations of primary and standby (active, master, follower, source, target, provider, consumer etc etc) that are used interchangeably everywhere across docs.